lib/mzgl/gl/Graphics.cpp

```diff
@@ -527,12 +321,11 @@ void Graphics::drawCircle(glm::vec2 c, float r) {
 	drawCircle(c.x, c.y, r);
 }
 void Graphics::drawCircle(float x, float y, float r) {
-	vector<glm::vec2> verts;
-	//vector<glm::vec4> cols;
+	std::vector<glm::vec2> verts;
 
 	int circleResolution = 100;
 	verts.reserve(circleResolution + 2);
-	//verts.push_back(glm::vec2(x, y));
+
 	for (int i = 0; i <= circleResolution; i++) {
 		float phi = M_PI * 2.f * i / (float) circleResolution;
 		verts.emplace_back(x + cos(phi) * r, y + sin(phi) * r);
```

Comment:
In fact, if `circleResolution` was a constexpr, you could then build a LUT of `phi` and avoid that computation. You could build an static constexpr array of std::pair(cos(phi), sin(phi)) and then this function would become 2 adds, 2 multiply and an emplace_back

Reply:
apparently trig functions aren't constexpr-able - but I can make it static so at least it only happens once